tests/plugins/subcommands/doctor/test_health_checks.py

```diff
@@ -245,6 +247,20 @@ def test_not_env_txt_check_action(
     assert X_MARK in captured.out
 
 
+def test_requests_ca_bundle_check_action(
+    env_ok: tuple[Path, str, str, str, str], capsys
+):
+    prefix, _, _, _, _ = env_ok
+    os.environ["REQUESTS_CA_BUNDLE"] = "non/existent/path"
```

Comment:
Let's use `monkeypatch` here instead in order to properly cleanup after the test completes